frontend/src/e2e-test/pages/employee/fridge-head-information/fridge-head-information-page.ts

```diff
@@ -3,12 +3,26 @@
 // SPDX-License-Identifier: LGPL-2.1-or-later
 
 import { Selector, t } from 'testcafe'
-import { formatCents } from 'employee-frontend/utils/money'
 import {
   selectFirstComboboxOption,
   waitUntilScrolled
 } from '../../../utils/helpers'
 
+function formatCents(amount?: number): string | undefined {
+  if (amount === undefined || amount === null) {
+    return undefined
+  }
+
+  const euros = amount >= 0 ? Math.floor(amount / 100) : Math.ceil(amount / 100)
+  const cents = Math.abs(amount % 100)
+
+  if (cents !== 0) {
+    return `${euros},${cents.toString().padStart(2, '0')}`
+  } else {
+    return euros.toString()
+  }
+}
+
```

Comment:
I think it would make more sense to move this function to lib-common.

Reply:
Done 👍